tests/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Project:  MapCache
+# Purpose:  MapCache tests
+# Author:   Even Rouault
```

Comment:
Jerome Boué

Reply:
Actually, I mainly copied tests/run_tests.sh from Even.
I intend to remove the original file once everyone agrees to remove Travis CI.